### ISSUE TITLE
ag-grid 24.0.0 (Angular) requires Peer Dependency of Angular 9 packages

### ISSUE THREAD

**Issue by nzacca** [ag-grid/ag-grid#4110]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of ag-Grid'
[] support request => see 'Requesting Community Support'
```

**Current behavior**
When installing ag-grid into an Angular 10 project, the cli produces a warning for a peer dependency of `@angular/common^9.0.0` and `@angular/core^9.0.0`.

**Expected behavior**

Expect that v24.0.0 can be installed in an Angular 10 project without a Peer Dependency warning.

Issue appears to stem from here:
https://github.com/ag-grid/ag-grid/blob/master/community-modules/angular/dist/ag-grid-angular/package.json#L7

Docs indicate that Angular 10 is supported by v24: https://www.ag-grid.com/angular-grid/#compatibility

Suggested solution:

Add a minimum requirement of Angular 9 `>=9.0.0`.

**Please tell us about your environment:**

Installed in Windows 10 using npm 6.14.3, node 12.16.1. Angular 10.1.4 project.

* **ag-Grid version:** 24.0.0

* **Browser:**
N/A

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
N/A

**Comment by makinggoodsoftware** [comment]
This is fixed in 24.1.0 which just been released.

**Comment by nzacca** [comment]
I have confirmed 24.1.0 has fixed this issue for us. Thank you very much!
